internal/cmd/deploy.go

```diff
@@ -13,8 +13,10 @@ type deployCommand struct {
 	cmd  *cobra.Command
 	args server.DeployArgs
 
-	tls        bool
-	tlsStaging bool
+	tls                bool
+	tlsStaging         bool
+	tlsCertificatePath string
+	tlsPrivateKeyPath  string
```

Comment:
We can use the members of the `ServiceOptions` directly, so don't need to add these here. Similar to how we're handling the `ErrorPagePath`.

(The other TLS variables here are because they don't exactly match the format that's in the `ServiceOptions` struct, so they get handled a bit differently).